patches/content-child-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/content/child/BUILD.gn b/content/child/BUILD.gn
+index 39264fc8a19083aeee6c65e1b4f3dd4d4da72aea..4b2bad00f3a4365eb7967eba2a73a228dedeced7 100644
+--- a/content/child/BUILD.gn
++++ b/content/child/BUILD.gn
+@@ -66,6 +66,7 @@ target(link_target_type, "child") {
+   deps = [
+     "//base",
+     "//build/config/compiler:compiler_buildflags",
++    "//brave/components/playlist:playlist_features",
```

Comment:
How about doing this with `sources.gni` like below?
With that, we don't need to touch this patch anymore when we want to add more deps or sources to this target in the future.
https://github.com/brave/brave-core/blob/master/patches/chrome-browser-ui-color-BUILD.gn.patch

Reply:
Sounds good! Refactored.